lib/nexus_cli/tasks.rb

```diff
@@ -5,6 +5,7 @@ module NexusCli
   module Tasks
     def self.included(base)
       base.send :include, ::Thor::Actions
+      # noinspection RubyInstanceMethodNamingConvention
```

Comment:
I see a few of these comments, but my Google searches have been coming up empty. Could you tell me what these do?